jina/peapods/pods/k8s.py

```diff
@@ -65,8 +65,12 @@ def port_expose(self) -> int:
 
     @property
     def host(self) -> str:
-        """Not implemented"""
-        raise NotImplementedError
+        """Currently, when deploying on Kubernetes, Jina does not expose a public host.
+        Instead Jina sends requests via port-forward and runs the requests against localhost.
+
+        :return: localhost
+        """
+        return 'localhost'
```

Comment:
Thats not correct I think. Its only localhost from the perspective of the port forwarding, but this property might be used in other contexts as well.

Reply:
What would you suggest?
In theory, there could be multiple ingress setup for the same gateway, so it could be even a list of hosts.